File: easy/118 (Pascal's Triangle)/solution.java
class Solution {
    public List<List<Integer>> generate(int numRows) {
        List<List<Integer>> pascals = new ArrayList<>();
        int counter = 0;
        for(int i = 0; i < numRows; i++){
            pascals.add(new ArrayList<>());
            pascals.get(i).add(1);
            for(int y = 1; y<counter; y++){
                int value = pascals.get(i - 1).get(y - 1) + pascals.get(i - 1).get(y);
                pascals.get(i).add(value);
            }
            if(i != 0 ) pascals.get(i).add(1);
            counter++;
        }
        return pascals;
    }
}

File: easy/118 (Pascal's Triangle)/solution.py
class Solution:
    def generate(self, numRows: int) -> List[List[int]]:
        pascals = []
        count = 0
        for i in range(0, numRows):
            pascals.append([])
            pascals[i].append(1)
            for y in range(1, count):
                pascals[i].append(pascals[i - 1][y - 1] + pascals[i - 1][y])

            if(i > 0):
                 pascals[i].append(1)
            count += 1
        return pascals